worlds/animal_well/region_data.py

```diff
@@ -448,18 +449,18 @@ class AWData(NamedTuple):
             AWData(AWType.region),  # wall is flush, just hold left
         rname.bear_truth_egg_spot:
             # fall down the shaft, catch yourself on a bubble, and jump right quickly before the bird pops it
-            AWData(AWType.region, [[iname.wheel_hard], [iname.bubble, iname.weird_tricks]]),
+            AWData(AWType.region, [[iname.wheel_hard], [iname.bubble, iname.precise_tricks]]),
         # top_of_the_well:  # unnecessary because of the connection from match center spot
         #     AWData(AWType.region, [[iname.bubble_long]]),
         rname.bear_upper_phone_room:
             AWData(AWType.region, [[iname.slink, iname.yoyo],
-                                   [iname.slink, iname.ball, iname.weird_tricks],
+                                   [iname.slink, iname.ball, iname.ball_trick_easy],
                                    # throw the ball in the yoyo pipe then run left with yoyo or slink
-                                   [iname.yoyo, iname.ball, iname.weird_tricks]]),
+                                   [iname.yoyo, iname.ball, iname.ball_trick_easy]]),
     },
     rname.bear_upper_phone_room: {
         rname.bear_above_chameleon:
-            AWData(AWType.region, [[iname.yoyo], [iname.ball, iname.weird_tricks]]),
+            AWData(AWType.region, [[iname.yoyo], [iname.ball, iname.ball_trick_medium, iname.obscure_tricks]]),
```

Comment:
This one probably has some more valid combinations -- I'd imagine slink would help?

Reply:
Yeah, actually. There was another way we found how to do this that was brought up in the Discord chat, but it didn't ever get pinned so I forgot about it. obscure_tricks + b.wand + slink. I'll commit it.
![image](https://github.com/user-attachments/assets/939e4c60-d71f-40ab-8eb4-3d352551a7c7)

Reply:
I'm sure there are more ways to do this, all in varying degrees of legitimacy, but I'm gonna let the community document them as they come up.